src/Client/Core/DurableTaskClientOptions.cs

```diff
@@ -46,6 +46,12 @@ public DataConverter DataConverter
         }
     }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether this client supports entities. If true, all instance ids starting with '@' are reserved for entities,
+    /// and validation checks are performed where appropriate.
+    /// </summary>
+    public bool SupportEntities { get; set; }
```

Comment:
What do you think of `EnableEntitySupport`?

Reply:
yes, slightly better I think.

Reply:
I have changed it for both `DurableTaskClientOptions` and `DurableTaskWorkerOptions`.